package-lock.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
@isunitha98selvan IMHO the decision whether to use a `package-lock.json` should be first tracked in a separate issue, and them this file should be added accordingly.
Please remove this file and create an issue to discussion addition of this file, and then add it via a separate PR based on the decision reached. Also, since this is a feature of npm 5 if I know correctly, we need to specify somewhere in the doc to make sure contributors are at npm5+ version. Please look into this.